### PR TITLE
Fixes for older browsers

### PR DESCRIPTION
**Pull request by jinty**
I hit these 2 issues on Safari 9. Trying to merge to master this time.

### CODE REVIEW

File: plugins/rias/ls.rias.js [hunk 1]
```diff
@@ -83,9 +83,20 @@
 		};
 
 		setOption = function(attr, run){
-			var attrVal = elem.getAttribute('data-'+ attr) || elemStyles.getPropertyValue('--ls-' + attr).trim() || null;
```
Comment (aFarkas) [inline plugins/rias/ls.rias.js]: You can make this a lot shorter: `(elemStyles.getPropertyValue('--ls-' + attr) || '').trim()`

File: plugins/rias/ls.rias.js [hunk 2]
```diff
@@ -186,6 +186,9 @@
 
 		if(buggySizes && elem.nodeName.toUpperCase() == 'IMG'){
 			elem.removeAttribute(config.srcsetAttr);
+			// at least on Safari 9, it looks like if srcset is set, it is preferred.
+			// Even though buggySizes is true. let's remove it to make sure src is used.
+			elem.removeAttribute('srcset');
```
Comment (aFarkas) [inline plugins/rias/ls.rias.js]: I don't want to take this right now. There are many different versions of Safari 9.x.x and many of them had different bugs. Removing `srcset` does too much. 

I assume you tested with a very early 9.0.x version that always takes the first image candidate, if it does not understand its content?

**Comment by aFarkas** [comment]
@jinty 
Do you want to split these changes into 2 PRs? I want to merge the first one as soon as possible to do a bugfix release.

**Comment by jinty** [comment]
On Tue, Oct 18, 2016 at 06:27:14AM -0700, Alexander Farkas wrote:
> @jinty 
> Do you want to split these changes into 2 PRs? I want to merge the first one as soon as possible to do a bugfix release.

Sorry,

don't have much time right now. I just removed the second commit so you
can merge the fix as is.

Though feel free to just do the first fix yourself with your preferred
syntax, I'm not really a JS programmer...

> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/aFarkas/lazysizes/pull/315#issuecomment-254506915

-- 
Brian Sutherland

**Comment by aFarkas** [comment]
Thanks about your other issue. Please try the following plugin: https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/fix-ios-sizes

**Pull request merged**